.github/workflows/action-lint-actions.yaml

```diff
@@ -0,0 +1,19 @@
+name: reviewdog
+on: [pull_request]
```

Comment:
You can specify the workflow to run only if the pull request is on actions. See one of the test workflows in this repo for inspiration (relevant to action-lint-workflows file too)

Reply:
Changed.